bin/main.rb

```diff
@@ -1,51 +1,46 @@
 #!/usr/bin/env ruby
 
-class Game
-  attr_accessor :counter
-  puts 'your turn...'
-  def initialize
-    puts 'Tell me your name...'
-    @player1 = gets.chomp
-    puts 'And your opponent name...'
-    @player2 = gets.chomp
-    @counter = 1
-  end
-
-  def start
-    result = 'sometext'
-    while result != 'victory' || result != 'draw'
-
-      puts 'Make your choice...'
-      board.mark_choice(gets.chomp)
-      board.display_board
-      @counter = 0 if @counter != 0
-      @counter = 1 if @counter == 0
-    end
-    puts result
-  end
-end
+board = ''
+result = ''
+
+# welcome users to game and asks for names
+
+puts "\nHello. Welcome to Tic Tac Toe Game.\n\n Tell me your name..."
+player_1 = gets.chomp
+puts "\n Okey, #{player_1}. Now tell me your opponent name..."
+player_2 = gets.chomp
+puts "\n Okey, #{player_1} & #{player_2}. {Some explanation on how to play goes here...a}Lets play.... "
+
+# use sleep to create time on user interface changes
+sleep 2.0
+
+#  here display empy board : board
+
+# keep on while if 'result' is different from victory or draw, exits when get some of them
+while result != 'victory' || result != 'draw'
+
+  puts 'Mark your choice...'
+  input_choice = gets.chomp.downcase
```

Comment:
This is not leading anywhere in the user interface, even thought it's a mock, it should work on it's surface:

```

Hello. Welcome to Tic Tac Toe Game.

 Tell me your name...
santiago

 Okey, santiago. Now tell me your opponent name...
pedro

 Okey, santiago & pedro. {Some explanation on how to play goes here...a}Lets play.... 
Mark your choice...
a

Mark your choice...
O

Mark your choice...
1

Mark your choice...
2

Mark your choice...
X

Mark your choice...
a)        

Mark your choice...
???   

Mark your choice...

```